server/grpc/api/service.go

```diff
@@ -24,6 +25,9 @@ func NewServiceServer(sdk *sdk.SDK) *ServiceServer {
 // Create creates a new service from definition.
 func (s *ServiceServer) Create(ctx context.Context, request *protobuf_api.CreateServiceRequest) (*protobuf_api.CreateServiceResponse, error) {
 	definition := fromProtoService(request.Definition)
+	if err := service.ValidateService(definition); err != nil {
```

Comment:
Should the Engine calls `service.ValidateService` in api package or directly in sdk package?

Reply:
PR https://github.com/mesg-foundation/engine/pull/1113 add the validation in the serviceSDK.create function but only before the save, so it doesn't actually check the inputs before reading them.
So both make sense. Let's improve this later.

Reply:
In sdk package. The server package should handle validation on `definition.Service` level , validation for request and response transformation

Reply:
I removed the validation from here